src/service/url-replacements-impl.js

```diff
@@ -76,7 +88,7 @@ export class UrlReplacements {
     this.initialized_ = true;
     // Returns a random value for cache busters.
     this.set_('RANDOM', () => {
-      return Math.random();
+      return String(Math.random());
```

Comment:
Yes to expanding the type set. Not sure what closure would allow here, but I'd push it wider.